assets/js/components/notifications/AdSenseAlerts.js

```diff
@@ -53,41 +53,42 @@ function AdSenseAlerts() {
 
 	return (
 		<Fragment>
-			{ notifications.map( ( notification ) => (
-				<BannerNotification
```

Comment:
This notification has been updated to use up-to-date properties for notifications that the AdSense module actually returns. All legacy properties and properties that are not used anymore have been removed.

https://github.com/google/site-kit-wp/blob/60a3fed17f33b5a0aeb5e9795d0ff8b1cdbcc5a8/includes/Modules/AdSense.php#L338-L349